server/src/main/java/org/opensearch/common/cache/tier/TieredSpilloverCache.java

```diff
@@ -54,6 +55,9 @@ public class TieredSpilloverCache<K, V> implements ICache<K, V>, StoreAwareCache
      */
     private final List<StoreAwareCache<K, V>> cacheList;
 
+    // A list of policies to apply to values before allowing them into the disk tier.
+    private final List<CacheTierPolicy<V>> policies;
```

Comment:
Is it a valid use case to update this `policies` List during run time?

Reply:
I think this might be a valid use case in the future if we have more policy types. We had been thinking that caches using policies would define them at the cache level, and the user would be able to disable them or change their parameters during runtime with cluster settings. For example the user can effectively disable the took time policy by setting the threshold to 0. But we will think about this.